resources/windows/msi/service.mcollective.wxs

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <DirectoryRef Id="TBD">  # The ID has to match the Ruby Bin directory
+      <Component Id='MCOService' Guid="7601FCEA-90B3-CC69-6A69-4087FBC7292D" Win64="$(var.Win64)">
```

Comment:
We don't have access to `$(var.Win64)` since we are no longer using `puppet.wxi`